src/NexusMods.Library/InstallLoadoutItemJobWorker.cs

```diff
@@ -34,8 +34,11 @@ protected override async Task<JobResult> ExecuteAsync(InstallLoadoutItemJob job,
 
         var result = await ExecuteInstallersAsync(job, installers, cancellationToken);
 
-        if (!result.HasValue && job.Installer is not AdvancedManualInstaller)
+        if (!result.HasValue)
         {
+            if (job.Installer is not AdvancedManualInstaller)
+                return JobResult.CreateFailed($"Advanced installer did not succeed for `{job.LibraryItem.Name}` (`{job.LibraryItem.Id}`)");
```

Comment:
```suggestion
            if (job.Installer is AdvancedManualInstaller)
                return JobResult.CreateFailed($"Advanced installer did not succeed for `{job.LibraryItem.Name}` (`{job.LibraryItem.Id}`)");
```

Reply:
If the provided installer was the advanced installer, then we don't want to run it again if it failed/user cancelled.